http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/HttpServletWithSessionListenerIT.java

```diff
@@ -51,6 +57,30 @@ public void sessionSecured() {
         thenCheckActiveSessionsEqualTo(1);
     }
 
+    @Test
+    @DisplayName("undertow servlet test http2")
+    public void underTowHttp2Servert(VertxTestContext vertxTestContext) {
+        final String HTTP_2_VERSION = "HTTP_2";
+        Vertx vertx = Vertx.vertx();
+        String uriUnderTow = "/app/servlet/hello";
```

Comment:
1. `UnderTow`?
2. I'm not sure what is a purpose of assign a String to a variable and using it exactly once. But that's a side note, I respect your preference and you shouldn't feel obliged to change it.